mods/d2k/sequences/infantry.yaml

```diff
@@ -35,24 +43,24 @@ light_inf:
 		Facings: -8
 		Transpose: true
 	die1: DATA.R8
-		Frames: 382, 389, 396, 403, 410, 417, 424, 431, 438, 439, 440, 441
+		Frames: 382, 389, 396, 403, 410, 417, 424, 431, 438, 443, 448, 453
```

Comment:
This doesn't look right.  The frames in this sequence increment by 7, and it wouldn't make sense for the original game to change stride half way through the animation.  By that logic these should be using frames 445, 452, 459.